lux_header    header

#Ibeo Lux object data
uint64        scan_start_time
uint16        num_of_objects
object_list[] object     
